scripts/dictcli.py

```diff
@@ -245,14 +245,15 @@ def remove_old(languages):
 
 def check_root():
     """Ask for confirmation if running as root when unnecessary."""
-    if spell.can_use_data_path() and os.geteuid() == 0:
-        print("You're running Qt >= 5.10 which means qutebrowser will "
-              "load dictionaries from a path in your home-directory. "
-              "Unless you run qutebrowser as root (bad idea!), you "
-              "most likely want to run this script as your user. ")
-        answer = input("Do you want to continue anyways? [y/N] ")
-        if answer not in ['y', 'Y']:
-            sys.exit(0)
+    if sys.platform not in "win32":
```

Comment:
Do you need to use contains here? If python returns "in" as the system it will not go in this loop, which I don't think is what you intended.

Reply:
I'm not sure what you mean. `in` is a keyword and the specification for sys.platform doesn't show "in" as one of the possible returned strings.

Reply:
Sure, but if `in` is the hot new operating system in 5 years, if python is still maintained, it will.

I don't see why you're using "win32".contains(sys.platform) instead of "win32".equals(sys.platform)

Reply:
because even if I use "equals" and some non-unix OS joins the markets I will still need to add it to a list

Reply:
Sure, but there's no list here, there's only strings.
Maybe: `sys.platform not in {"win32"}`?

Reply:
This currently checks if the platform is e.g. "w", "i", "n" or something like "wi" etc. (it does substring matching) which definitely doesn't make sense.

Just use ==/!= instead - if any platform needs to be added in the future, this still can be changed to a list/set.

Also, please use an early return on Windows instead of placing the entire function under the "if".